sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/GalleriesOperations.cs

```diff
@@ -134,7 +138,7 @@ public async Task<AzureOperationResponse<Gallery>> UpdateWithHttpMessagesAsync(s
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, string select = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Need a customization to fix this breaking change - essentially you need a method that implements the old parameters, with 'customHeaders' being non-optional, like the following:

```
public partial class GalleriesOperations {

    public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, Dictionary<string, List<string>> customHeaders, CancellationToken cancellationToken = default(CancellationToken))
    {
         return GetWithHttpMessagesAsync(resourceGroupName, galleryName, null, customHeaders, cancellationToken);
    }
}
```